RecoLocalTracker/Phase2TrackerRecHits/src/Phase2StripCPEGeometric.cc

```diff
@@ -0,0 +1,28 @@
+
+#include "RecoLocalTracker/Phase2TrackerRecHits/interface/Phase2StripCPEGeometric.h"
+#include "SimTracker/SiPhase2Digitizer/plugins/Phase2TrackerDigitizerFwd.h"
```

Comment:
I just removed the dependency, the PR is updated. I kept the typedefs but inlined in the code, to have in one single place this at first sight weird choice for the phase2 outer-tracker geomdetunit and topology classes.